pyproject.toml

```diff
@@ -48,6 +48,7 @@ h5py = "^3.2"
 lxml = "^4.6"
 lasio = "^0.29"
 scipy = "^1.7.3"
+numba = "^0.55.0"
```

Comment:
This states that we require the very latest version of numba, minor version `0.55` and nothing else. Can we not have a more flexible constraint, e.g. `numba = "^0.50` ?

This will allow compatibility with recent minor versions and will also allow us to keep up with future non-breaking updates?

Reply:
That is the minimum requirement, otherwise it doesn't satisfy the Python version constraints

Reply:
Could you clarify, which python version constraints would be incompatible with e.g. numba `v0.50` ?

Reply:
Thanks for the discussion Max and for testing the other versions! Looks like we should go for `^0.52`.